lib/cext/ruby/rubyio.h

```diff
@@ -0,0 +1 @@
+#include "../ruby.h"
```

Comment:
Note that this header is defined under `ruby/backward` in MRI, but hopefully it doesn't matter.